package/src/components/ChannelList/hooks/usePaginatedChannels.ts

```diff
@@ -36,6 +37,29 @@ const DEFAULT_OPTIONS = {
   message_limit: 10,
 };
 
+const offlineSort = <
+  At extends UnknownType = DefaultAttachmentType,
+  Ch extends UnknownType = DefaultChannelType,
+  Co extends string = DefaultCommandType,
+  Ev extends UnknownType = DefaultEventType,
+  Me extends UnknownType = DefaultMessageType,
+  Re extends UnknownType = DefaultReactionType,
+  Us extends UnknownType = DefaultUserType,
+>(
+  channels: Channel<At, Ch, Co, Ev, Me, Re, Us>[],
+  sortObj: ChannelSort<Ch>,
+) => {
+  const sortKeys: string[] = [];
+  const sortOrders: Array<boolean | 'asc' | 'desc'> = [];
+
+  Object.keys(sortObj).forEach((key) => {
+    sortKeys.push(`state.${key}`);
```

Comment:
im not 100% sure about this because it seems like the backend sort filters are a bit different? like here i can only access last_message_at in the `channel.state`. but what about other properties?